docs/testrunner-toolkit/configuration/common-syntax.md

```diff
@@ -171,10 +171,34 @@ __Example__:
       lodash: "4.17.20"
       "@babel/preset-typescript": "7.12"
       "@cypress/react": "^5.0.1"
-      
 ```
 
-⚠️ `registry` configuration is only supported in latest cypress docker image. No other frameworks, or Sauce Labs cloud.
+### `registry`
+
+__Description__: Specifies npm registry source.
+
+__Type__: *string*
+
+__Example__:
+```yaml
+  registry: https://registry.npmjs.org
+```
+
+⚠️ `registry` can also be private internal npm registry source. It works along with [Sauce Connect](/testrunner-toolkit/running-tests#run-tests-on-sauce-labs-with-sauce-connect) to get access to your private internet from Sauce Cloud.
```

Comment:
Hey @tianfeng92  -- I am sorry I didn't catch this before. Have you manually inserted a caution symbol here? Typically, we use the admonition syntax here to put in a warning or caution alert. Is this statement something about which customers should be cautioned? What is the risk level? Thanks and sorry again that I missed this before.

Reply:
I didn't set this by myself. It was existed previously. I can remove this and add a note section

Reply:
Huh. When was it added? It is not in the current doc?